app/routes/lending/components/LendingRequestAdmin.tsx

```diff
@@ -0,0 +1,187 @@
+import FullCalendar from '@fullcalendar/react';
+import timeGridPlugin from '@fullcalendar/timegrid';
+import { LoadingIndicator } from '@webkom/lego-bricks';
+import { usePreparedEffect } from '@webkom/react-prepare';
+import { Helmet } from 'react-helmet-async';
+import { Link, useParams } from 'react-router-dom';
+import {
+  fetchLendingRequest,
+  fetchLendingRequestsForLendableObject,
+} from 'app/actions/LendingRequestActions';
+import {
+  Content,
+  ContentMain,
+  ContentSection,
+  ContentSidebar,
+} from 'app/components/Content';
+import InfoList from 'app/components/InfoList';
+import NavigationTab, { NavigationLink } from 'app/components/NavigationTab';
+import { FromToTime } from 'app/components/Time';
+import {
+  selectLendingRequestById,
+  selectLendingRequestsByLendableObjectId,
+} from 'app/reducers/lendingRequests';
+import { useAppDispatch, useAppSelector } from 'app/store/hooks';
+
+type Params = {
+  lendingRequestId: string;
+};
+
+const LendingRequestAdmin = () => {
+  const { lendingRequestId } = useParams<Params>();
+  const dispatch = useAppDispatch();
+
+  usePreparedEffect(
+    'fetchRequest',
+    () => dispatch(fetchLendingRequest(Number(lendingRequestId))),
+    [],
+  );
+
+  const request = useAppSelector((state) =>
+    selectLendingRequestById(state, Number(lendingRequestId)),
+  );
+
+  const fetching = useAppSelector((state) => state.lendableObjects.fetching);
+
+  usePreparedEffect(
+    'fetchRequests',
+    () => {
+      if (request && request.lendableObject?.id) {
+        dispatch(
+          fetchLendingRequestsForLendableObject(request.lendableObject.id as number),
+        );
+      }
+    },
+    [request?.lendableObject.id],
+  );
+
+  const otherRequests = useAppSelector((state) =>
+    selectLendingRequestsByLendableObjectId(state, {
+      lendableObjectId: request?.lendableObject.id,
+    }),
+  );
+
+  const otherLoans = otherRequests.filter((loan) => !loan.pending);
+  const otherLoanRequests = otherRequests.filter((loan) => loan.pending);
+
+  if (!request) {
+    return <p>Ukjent forespørsel</p>;
+  }
+
+  const requestEvent = {
+    id: String(request.id),
+    title: request.author?.fullName,
+    start: request.startDate,
+    end: request.endDate,
+    backgroundColor: '#e11617',
+    borderColor: '#e11617',
+  };
+
+  const otherLoanEvents = otherLoans.map((loan) => ({
+    id: String(loan.id),
+    title: request.author?.fullName,
+    start: loan.startDate,
+    end: loan.endDate,
+    backgroundColor: '#999999',
+    borderColor: '#999999',
+  }));
+
+  const otherLoanRequestEvents = otherLoanRequests.map((loan) => ({
+    id: String(loan.id),
+    title: request?.author?.fullName,
+    start: loan.startDate,
+    end: loan.endDate,
+    backgroundColor: '#f57676',
+    borderColor: '#f57676',
+  }));
+
+  const infoItems = [
+    {
+      key: 'Status',
+      value: request.pending ? 'Venter på svar' : 'Godkjent',
+    },
+    {
+      key: 'Tidspenn',
```

Comment:
```suggestion
      key: 'Tidsspenn',
```

Perhaps reuse the other `infoItems`?